### ISSUE TITLE
Exception in Freemarker template is logged but no response is returned (hangs)

### ISSUE THREAD

**Issue by geertvanheusden** [micronaut-projects/micronaut-views#446]
### Expected Behavior

If an exception is thrown in the Freemarker template (e.g. `freemarker.core.ParseException`) an Internal Server error (500) is returned in the HttpResponse.

It is probably related to this ticket: https://github.com/micronaut-projects/micronaut-views/issues/91 which was closed without a solution as far as I can see.

### Actual Behaviour

The error is logged but the response "hangs" and is never returned.
The exception can not be caught by an ErrorHandler.

### Steps To Reproduce

Create a freemarker template with an incorrect syntax (e.g. `${()}`) and a controller that renders this template:

```
@View("test.ftl")
@Get("/test")
public Mono<HttpResponse> test() {
     return Mono.just(HttpResponse.ok());
}
```

### Version

micronaut-core:3.7.0, micronaut-views-core: 3.6.0

**Comment by sdelamo** [comment]
is this fixed @wetted ? I think we did something similar

**Comment by wetted** [comment]
> is this fixed @wetted ? I think we did something similar

@sdelamo yes, I fixed this on #501 for `micronau-views` v3.8.x, but it doesn't look like that has been merged up to 3.9.x or master yet.
